client_react/src/_constants/text.constants.js

```diff
@@ -128,6 +140,16 @@ export const resources = {
             CREATE: 'Создать',
             ADD: 'Добавить',
             PHONE_NUMBER: 'Номер телефона',
+            FORGOT_PASSWORD_TITLE: 'Сменить пароль',
+            REQUEST_FORGOT_PASSWORD: 'Запросить смену пароля',
```

Comment:
Where would the user see this message? It sounds a bit confusing.

Reply:
On the button